src/api/route-services/experiment.js

```diff
@@ -4,11 +4,13 @@ const mockData = require('./mock-data.json');
 const AWS = require('../../utils/requireAWS');
 const logger = require('../../utils/logging');
 
+const { OK, NotFoundError } = require('../../utils/responses');
+
 const {
   createDynamoDbInstance, convertToJsObject, convertToDynamoDbRecord, configArrayToUpdateObjs,
 } = require('../../utils/dynamoDb');
 
-const { NotFoundError } = require('../../utils/responses');
+const adminArn = '70c213d4-e7b6-4920-aefb-706ce8606ee2';
```

Comment:
Make this a constant syntax like `ADMIN_ARN`. Also, there are two ARNs, one for `staging` and one for `production`. Add both of them, or have a function that chooses depending on the environment we are in.